ec2api/api/__init__.py

```diff
@@ -193,6 +196,14 @@ def __call__(self, req):
         # calculation will yield invalid results.
         body_hash = hashlib.sha256(req.body).hexdigest()
 
+        # Verify the version is as expected from config file
+        req_version = req.params.get('Version')
+        if not req_version or req_version not in CONF.supported_api_versions:
+            _msg = ("Unsupported Version used in the request. Supported "
+                    "versions are " + str(CONF.supported_api_versions))
```

Comment:
Can we not send "supported versions are". It should be pretty clear in the documentation. We should not expose it from our side.